dataModel.NeMoBil/Skill/schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "required" : [ "id", "type" ],
+  "type" : "object",
+  "properties" : {
+    "id" : {
```

Comment:
1) format int32 is not a valid json schema, adn 
2) The id is usually included in the GSMA properties with the valid options 
By default we include this 
  "allOf": [
    {
      "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/GSMA-Commons"
    },
    {
      "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
    },
    That includes attributes for the regular use of a data model and also for location, and an additional clause with properties